til-domain/src/main/java/com/til/domain/category/repository/AdminCategoryRepositoryCustomImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.til.domain.category.repository;
+
+import java.util.List;
+
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import com.til.domain.category.enums.CategoryErrorCode;
+import com.til.domain.category.model.QCategory;
+import com.til.domain.common.exception.BaseException;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class AdminCategoryRepositoryCustomImpl implements AdminCategoryRepositoryCustom {
+
+    private final JPAQueryFactory queryFactory;
+
+    @Override
+    public void validateCategoryIds(List<Long> categoryIdList) {
+        QCategory category = QCategory.category;
+
+        long validCount = queryFactory.select(category.id)
+            .from(category)
+            .where(category.id.in(categoryIdList))
+            .fetchCount();
```

Comment:
P3: fetchCount()가 deprecated 되어서 아래와 같이 변경해보면 좋을 것 같습니다!
```suggestion
        long validCount = queryFactory.select(category.count())
            .from(category)
            .where(category.id.in(categoryIdList))
            .fetchFirst();
```